zmon_worker_monitor/builtins/plugins/elasticsearch.py

```diff
@@ -33,24 +33,29 @@ class ElasticsearchFactory(IFunctionFactoryPlugin):
     def __init__(self):
         super(ElasticsearchFactory, self).__init__()
 
+        self._url = None
+
     def configure(self, conf):
         """
         Called after plugin is loaded to pass the [configuration] section in their plugin info file
         :param conf: configuration dictionary
         """
-        return
+        self._url = conf.get('url')
```

Comment:
@Jan-M from what I see, it doesn't get injected, only the value in the config is used. Is it supposed to be auto-injected or should it be used in the check command (e.g. ``elasticsearch(entity['url'])...`` ) ?